service/project.go

```diff
@@ -451,8 +468,9 @@ func (uis *UIServer) modifyProject(w http.ResponseWriter, r *http.Request) {
 			responseRef.Triggers[i].DefinitionID = utility.RandomString()
 		}
 	}
-	for _, t := range responseRef.PatchTriggerAliases {
+	for i, t := range responseRef.PatchTriggerAliases {
 		catcher.Add(t.Validate(id))
+		responseRef.PatchTriggerAliases[i].ChildProject = t.ChildProject
```

Comment:
Without knowing that t.Validate modifies the Child Project, this just reads as t _is_ responseRef.PatchTriggerAliases[i], and so it isn't immediately clear that this is doing something, so I would drop a comment. (Or you may be able to call Validate directly on `responseRef.PatchTriggerAliases[i]` and so then you don't need to reassign anything)